magit-file-icons.el

```diff
@@ -88,11 +128,17 @@
    (magit-file-icons-mode
     (when magit-file-icons-enable-diff-file-section-icons (el-patch-eval-template #'magit-diff-insert-file-section 'defun))
     (when magit-file-icons-enable-untracked-icons (el-patch-eval-template #'magit-insert-untracked-files 'defun))
-    (when magit-file-icons-enable-diffstat-icons (el-patch-eval-template #'magit-diff-wash-diffstat 'defun)))
+    (when magit-file-icons-enable-diffstat-icons (el-patch-eval-template #'magit-diff-wash-diffstat 'defun))
+    (when magit-file-icons-enable-diff-file-section-icons (magit-file-icons-refresh-backend))
+    (when magit-file-icons-enable-untracked-icons (magit-file-icons-refresh-backend))
+    (when magit-file-icons-enable-diffstat-icons (magit-file-icons-refresh-backend)))
+    (add-hook 'magit-mode-hook 'magit-file-icons-refresh-backend)
```

Comment:
I think we can get rid of these hooks - it's probably sufficient to refresh on minor mode activation.